### PR TITLE
fix: Fix IDB random-access-storage driver reset

### PR DESCRIPTION
**Pull request by dmaretskyi**
### Motivation / Background

This Pull Request ...

### CODE REVIEW

File: packages/common/random-access-storage/src/common/storage.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ export enum StorageType {
   RAM = 'ram',
   IDB = 'idb',
   CHROME = 'chrome',
```
Comment (wittjosiah) [inline packages/common/random-access-storage/src/common/storage.ts:11]: what is "chrome" storage?

File: packages/common/random-access-storage/src/common/storage.ts [hunk 2]
```diff
@@ -9,6 +9,9 @@ export enum StorageType {
...
+  /**
+   * @deprecated
+   */
   FIREFOX = 'firefox',
   NODE = 'node',
   WEBFS = 'webfs',
```
Comment (wittjosiah) [inline packages/common/random-access-storage/src/common/storage.ts:17]: this is going to be deprecated too, right?

Comment (dmaretskyi) [reply packages/common/random-access-storage/src/common/storage.ts:17]: yes, eventually

File: packages/common/random-access-storage/src/common/storage.ts (hunk 1 above)
Comment (dmaretskyi) [reply packages/common/random-access-storage/src/common/storage.ts:11]: Doesn't seem to be used, we can deprecate this too

**Pull request merged**